test/unittest/configuration/ConfigurationTest.cpp

```diff
@@ -480,8 +480,9 @@ TEST(ConfigurationTest, allowlist_wildcard)
     add_topic_to_list_to_yaml(yaml3, ALLOWLIST_TAG, "topic2*", "type2*");
     DDSRouterConfiguration config3(yaml3);
     auto result3 = config3.allowlist();
-    EXPECT_TRUE(topic_in_list(result3, WildcardTopic("topic1", "type1")));
-    EXPECT_TRUE(topic_in_list(result3, WildcardTopic("topic2*", "type2*")));
+    EXPECT_FALSE(topic_in_list(result3, WildcardTopic("topic1", "type1", true)));
```

Comment:
I would leave the tests as they were and add a new test only for keyed and not keyed tests.
As not setting keyed in filter topics will work for both keyed and no_keyed, the current tests will keep being available.

Reply:
:+1: